core/Cargo.toml

```diff
@@ -64,6 +64,9 @@ sp-core = { workspace = true, optional = true }
 sp-runtime = { workspace = true, optional = true }
 tracing = { workspace = true, default-features = false }
 
+# AccountId20
+keccak-hash = { workspace = true}
```

Comment:
Can we hide the dependency `keccak-hash` behind the subxt-compat feature flag now?